sql/policy_internal.sql

```diff
@@ -154,6 +157,10 @@ BEGIN
          EXIT;
     END IF;
   END LOOP;
+
+  IF failure IS TRUE THEN
+    RAISE EXCEPTION 'compression policy failure';
```

Comment:
Probably good to add some more info in the exception. Something similar to:
`Compression policy failure: failed to compress 5 chunks. successfully compressed 7 chunks.`

Reply:
Make sense, will add it into the DETAIL.